docs/read_the_docs/source/advanced_tutorial/writing_housing_characteristics.rst

```diff
@@ -0,0 +1,12 @@
+Writing Housing Characteristics
+###############
+
+This section provides a description the standard format for the housing characteristics. In order to stop recommitting entire files and to keep differences easy to read during the review process, a standard format for writing housing characteristics has been created. All housing characteristics shall follow the following format guidelines.
+
+Guidelines
+=============
+
+1. All line terminators are '\\r\\n'.
+2. All Option columns have format '%.6f'.
+3. All [For Reference Only] Source Sample Size columns are type 'int'.
+4. All [For Reference Only] Source Weight columns are type 'int'.
```

Comment:
Add guidelines on "Created by" line

Reply:
- Row values should sum to 1.000000.
- All combinations of dependencies are required to have values

Reply:
Are you documenting what the integrity_checks do? There are additional checks (like [this one](https://github.com/NREL/OpenStudio-BuildStock/pull/313)).

Reply:
@ejhw @shorowit Once I have the integrity checks updated, I will make these updates.  Thank you for the review and providing an example.